base_layer/chat_ffi/src/lib.rs

```diff
@@ -69,6 +71,9 @@ mod consts {
     include!(concat!(env!("OUT_DIR"), "/consts.rs"));
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub struct ByteVector(Vec<c_uchar>); // declared like this so that it can be exposed to external header
```

Comment:
if it needs exposing, then it needs rep_c as well?

Reply:
I copied it verbatim from the wallet, which doesn't use rep_c for it.